lib/pengine/pe_output.c

```diff
@@ -649,12 +649,12 @@ ban_xml(pcmk__output_t *out, va_list args) {
     return pcmk_rc_ok;
 }
 
-PCMK__OUTPUT_ARGS("ban-list", "pe_working_set_t *", "const char *", "GList *",
+PCMK__OUTPUT_ARGS("ban-list", "pe_working_set_t *", "char *", "GList *",
                   "unsigned int", "gboolean")
 static int
 ban_list(pcmk__output_t *out, va_list args) {
     pe_working_set_t *data_set = va_arg(args, pe_working_set_t *);
-    const char *prefix = va_arg(args, const char *);
+    char *prefix = va_arg(args, char *);
```

Comment:
I'd rather make the top-level arguments const char*

Reply:
Fixed, though I will need to teach my fosa code about casts (which I thought was working, but...)